rtos/rtx5/TARGET_CORTEX_M/rtx_lib.c

```diff
@@ -595,7 +595,9 @@ void *__user_perthread_libspace (void) {
 typedef void *mutex;
 
 // Initialize mutex
+#if !defined(__ARMCC_VERSION) || __ARMCC_VERSION < 6010050
```

Comment:
Thanks @bulislaw. It looks like they're staying until the CMSIS guys can find a better solution.